Source/WebCore/platform/graphics/DecodingOptions.h

```diff
@@ -69,9 +78,12 @@ class DecodingOptions {
         return sizeForDrawing()->maxDimension() >= other.sizeForDrawing()->maxDimension();
     }
 
+    DecodeToHDR decodeToHDR() const { return m_decodeToHDR; }
```

Comment:
The name sounds imperative ("decode to HDR!") but maybe it's a `shouldDecodeToHDR()`.